Assets/00_Content/Scripts/Interactables/Kitchens/KitchenTicket.cs

```diff
@@ -0,0 +1,28 @@
+
+using UnityEngine;
+
+namespace Interactables.Kitchens {
+	public class KitchenTicket : PickUp {
+		[SerializeField] private float upVelocity;
+		[SerializeField, Min(0)]
+		private float minSpawnVelocity;
+		[SerializeField, Min(0)]
+		private float maxSpawnVelocity;
+
+		protected override void Start() {
```

Comment:
I'm guessing this is taken from coin. I do think it would be better move this, alongside coin throwing, into the source who actually throws it.

Reply:
I like this idea, but cointhrowing and ticketspawning is handled kinda differently. Can´t find a way to make this more generic with the current implementation sadly.